site/content/docs/user-guide/using-custom-indexes.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Using Custom Plugin Indexes
+slug: custom-indexes
+weight: 800
+draft: true
+---
+
+Plugin indexes contain plugin manifests, which are documents that describe
+installation procedure for a plugin. For discovery purposes, Krew comes with a
+`default` plugin index,  plugins hosted in the [`krew-index`
+repository](https://github.com/kubernetes-sigs/krew-index).
+
+However, some plugin authors may choose to host their own indexes that contain
+their curation of kubectl plugins. These are called "custom plugin indexes".
+
+## Adding a custom index
```

Comment:
Can you also add an "advanced" section, where you explain how users/vendors could substitute the krew default index with their own?

Reply:
Whoops I missed this comment when I was looking at your review yesterday. Sure I can add this